weave-js/src/common/components/WandbLoader.tsx

```diff
@@ -42,6 +42,10 @@ export interface TrackedWandbLoaderProps extends WandbLoaderProps {
   samplingRate?: number;
   /* Tell me you're a Segment .track() event without telling me about Segment */
   track: (eventName: string, data: Record<string, unknown> | undefined) => void;
+  /* Optional callback fired when finished loading */
+  onComplete?(name: string, data: Record<string, unknown> | undefined): void;
+  /* Optional callback fired when started loading */
+  onStart?(name: string): void;
```

Comment:
it looks like you're adding these but not using them anywhere...? i'm guessing you're saving it for a future PR?

just an aside, this is my issue with the tiny-PR movement... the reviewer has like no context on the full scope of the problem and planned solution.